packages/core/src/RepoSync.ts

```diff
@@ -202,255 +163,96 @@ export class RepoSync {
     }
   }
 
-  // Private methods
-
-  /**
-   * Called for each document upon initialization. Records the document's clock and advertises it.
-   * @param documentId
-   */
-  private async registerDoc(documentId: string) {
-    this.log('registerDoc', documentId)
-    const clock = await this.getClockFromDoc(documentId)
-
-    // Make sure we can sync this document
-    this.validateDoc(documentId, clock)
-
-    // Record the doc's initial clock
-    await this.updateClock(documentId, ours, clock)
-  }
-
-  /**
-   * Checks the local doc's clock to ensure that it can be synced
-   * @param documentId
-   * @param clock
-   */
-  private validateDoc(documentId: string, clock: Clock) {
-    this.log('validateDoc', documentId)
```

Comment:
"validation" wasn't really doing anything for us, since we control the documents' whole lifecycle.